docs/conf.py

```diff
@@ -73,9 +73,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = "v1.5"
+version = "v2.0"
 # The full version, including alpha/beta/rc tags.
-release = "v1.5.0-alpha"
+release = "v2.0.0-alpha"
```

Comment:
This is an example written during v1.5 dev cycle and is still correct in concept, the 'v1.5' here is actually just an example of the concepts, not a record of actual changes/tags. This can safely be ignored in this instance, albeit slightly out of date.